src/Microsoft.TemplateEngine.Cli/Commands/TemplateOption.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.CommandLine;
+
+namespace Microsoft.TemplateEngine.Cli.Commands
+{
+    internal class TemplateOption
+    {
+        private readonly Option _option;
+
+        internal TemplateOption(
+            CliTemplateParameter parameter,
+            IEnumerable<string> aliases)
+        {
+            TemplateParameter = parameter;
+            Aliases = aliases.ToArray();
```

Comment:
Not for this PR: would be nice if `AliasAssignmentCoordinator.AssignAliasesForParameter